libs/service-portal/endorsements/src/screens/Endorsements/Endorsements.tsx

```diff
@@ -0,0 +1,348 @@
+import React from 'react'
```

Comment:
isn't this something that has been removed on main?

Reply:
Yes correct! - Removed ✔️